### PR TITLE
Update prices API, add 'paid_from_trading_vol', extra enable fields

### PR DESCRIPTION
**Pull request by smk762**

### CODE REVIEW

File: docs/basic-docs/atomicdex-api-20/trade_preimage.md [hunk 1]
```diff
@@ -47,12 +47,13 @@ Use the `trade_preimage` request with `max = true` and `swap_method = "setprice"
...
+| amount                | string (numeric) | fee amount (in decimal representation)                                                                                                                        |
+| amount_rat            | rational         | fee amount (in rational representation)                                                                                                                       |
+| amount_fraction       | fraction         | fee amount (in fraction representation)                                                                                                                       |
+| paid_from_trading_vol | boolean          |  If `true`, fees are deducted from the payment amount for the spend/refund UTXO HTLC transaction. If `false`, fees are not deducted from the traded volume. This is either where maker side pays additional miner fee to broadcast their swap transaction, or where gas paid (e.g in ETH for an ERC20 trade) - in this case, user requires a sufficient current ETH balance to cover the fees before they can initiate the swap)                                             |
```
Comment (artemii235) [inline docs/basic-docs/atomicdex-api-20/trade_preimage.md:56]: > maker side pays additional miner fee

I used "maker" as an example - it applies to taker transactions (dex fee, swap payment) too. Could you change it please to reflect this meaning?

**Review by artemii235 — CHANGES_REQUESTED** [review]
One minor change.

File: docs/basic-docs/atomicdex-api-20/trade_preimage.md (hunk 1 above)
Comment (smk762) [reply docs/basic-docs/atomicdex-api-20/trade_preimage.md]: reworded, thanks.

**Review by artemii235 — APPROVED** [review]
LGTM, thanks!

**Comment by smk762** [comment]
@gcharang this PR includes a draft doc for coin activation, combining both methods as subsections and including examples for other platforms like BNB and MATIC. It's not in the sidebar yet so not exposed, but if you can confirm nothing is missing and I have the section ## hashes correct I'd like to add it to replace the current enable.md and electrum.md docs.

File: docs/basic-docs/atomicdex-api-legacy/coin_activation.md [hunk 2]
```diff
@@ -0,0 +1,499 @@
...
+| gas_station_policy.policy  | string (optional for ETH/ERC20 and other gas model chains) | Defines the method of gas price calculation from the station response. `"MeanAverageFast"` will use the mean between average and fast fields. `"Average"` will return a simple average value. Defaults to `"MeanAverageFast"`. |
+| mm2                    | integer                        | Required if not set in `coins` file. Informs the AtomicDEX API whether or not the coin is expected to function. Accepted values are `0` or `1`                                   |
+| tx_history                        | bool                                             | If `true` the AtomicDEX API will preload trasaction history as a background process. Must be set to `true` to use the [my_tx_history](https://developers.komodoplatform.com/basic-docs/atomicdex-api-legacy/my_tx_history.html#my-tx-history) method              |
```
Comment (gcharang) [inline docs/basic-docs/atomicdex-api-legacy/coin_activation.md:243]: use relative url

File: docs/basic-docs/atomicdex-api-legacy/coin_activation.md [hunk 3]
```diff
@@ -0,0 +1,499 @@
...
+</div>
+
+To see more examples for other platforms like Fantom, Avalanche & Harmony, you can search the [AtomicDEX API Coin Activation Commands List
+](http://stats.kmd.io/atomicdex/activation_commands/) or build a single `batch` command to enable a set of selected coins via the 
+[Batch Coin Activation Form](http://stats.kmd.io/atomicdex/batch_activation_form/)
+
+
+::: tip
+
+We welcome volunteers to test new coins for AtomicDEX API compatibility! Follow the [Submitting Coin Test Results guide](https://github.com/KomodoPlatform/coins#about-this-repository) for more information, or drop into the [Komodo Platform Discord Server] for a chat if you need some help.
```
Comment (gcharang) [inline docs/basic-docs/atomicdex-api-legacy/coin_activation.md:497]: `https://komodoplatform.com/discord`

**Review by gcharang — CHANGES_REQUESTED** [review]
Other than the comments I have left, rest looks good

File: docs/basic-docs/atomicdex-api-legacy/coin_activation.md (hunk 3 above)
Comment (smk762) [reply docs/basic-docs/atomicdex-api-legacy/coin_activation.md]: fixed

File: docs/basic-docs/atomicdex-api-legacy/coin_activation.md (hunk 2 above)
Comment (smk762) [reply docs/basic-docs/atomicdex-api-legacy/coin_activation.md]: fixed

File: docs/basic-docs/atomicdex-api-legacy/coin_activation.md [hunk 4]
```diff
@@ -0,0 +1,499 @@
...
+::: tip Note
+
+The AtomicDEX API node's coin address needs to be imported manually into the coin daemon using the [importaddress](../../../basic-docs/smart-chains/smart-chain-api/wallet.html#importaddress) method.
+
+:::
+
+
+## Lite mode activation
+
+Activating coins in 'lite mode' means you don't need to have a native coin daemon installed, or keep a local blockchain syncronised. In this mode, the AtomicDEX API communicates with an external node to perform transactions and query the blockchain. UTXO based coins and QTUM/QRC20 tokens communicate via electrum servers, while other platform coins communicate via JSON RPC urls.
```
[the remark this answers is not in the document]
Comment (smk762) [reply docs/basic-docs/atomicdex-api-legacy/coin_activation.md]: fixed

File: docs/basic-docs/atomicdex-api-legacy/coin_activation.md [hunk 5]
```diff
@@ -0,0 +1,499 @@
...
+
+There are two methods of coin activation:
+- `enable` - Connects to a native daemon, or a url which handles RPCs for Platform coins (e.g. ETH, MATIC, FTM, BNB, ONE)
+- `electrum` - Connects to an [electrum server](https://github.com/kyuupichan/electrumx) for UTXO based coins and QTUM/QRC20 tokens.
+
+A coin can only be activated once per session, and must be activated before it can be used in trading or wallet functions. 
+
+::: warning Important
+
+The AtomicDEX API requires an `mm2` parameter to be set for each `coin`. This can be added to the enable/electrum command, or defined in your [coins file](https://developers.komodoplatform.com/basic-docs/atomicdex/atomicdex-tutorials/atomicdex-walkthrough.html#setting-up-the-coin-list).
```
[the remark this answers is not in the document]
Comment (smk762) [reply docs/basic-docs/atomicdex-api-legacy/coin_activation.md]: fixed

**Comment by smk762** [comment]
thanks @gcharang - I've done the review updates, removed enable/electrum docs and added activation doc to sidebar.

**Review by gcharang — APPROVED** [review]

**Comment by gcharang** [comment]
thanks @smk762 ! looks good. once this PR is merged, I will add redirects for the old urls to point to their places in the new doc

**Pull request merged**